disperser/dataapi/subgraph_client.go

```diff
@@ -98,7 +98,7 @@ type (
 	}
 	BatchNonSigningInfo struct {
 		QuorumNumbers        []uint8
-		ReferenceBlockNumber uint64
+		ReferenceBlockNumber uint32
```

Comment:
nit: I think block numbers are usually typed as `uint64` (at least in geth). Why is it better to represent it as uint32 here?

Reply:
The convention so far is that both smart contract and offchain are using uint32. I'm fine with uint64 as well but we need to be consistent. The uint32 should be fine in practice as well.